torq.q

```diff
@@ -426,17 +426,17 @@ createalias:{[logdir;filename;alias]
 // makealias = if true, will create alias files without the timestamp value
 createlog:{[logdir;logname;timestamp;suppressalias]
 	basename:(string logname),"_",(string timestamp),".log";
-	alias:$[suppressalias;"";(string logname),".log"];
+	alias:$[suppressalias;"";(string logname),".log"];$[onelog;fileredirect[logdir;"err_out_",basename;"err_out_",alias;2];[
```

Comment:
could you stick a line break in here.  
if it's just a single log file could it not have a prefix - just the basename will do. 
Are you redirecting 1 correctly if there is a single log file?